src/utils/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
+
+export function useLocalStorage(key, initialValue) {
+  const [value, setValue] = useState(() => {
+    try {
+      const localValue = window.localStorage.getItem(key);
+      return localValue ? JSON.parse(localValue) : initialValue;
+    } catch (error) {
```

Comment:
as a general standard, try to add _ before arguments that you're not using yet. in this case it would be '_error', this is a common practice in JS. This value could be logged in the future but for now keep it visible by adding the underscore.